superset-frontend/src/dashboard/components/gridComponents/Row.jsx

```diff
@@ -201,7 +234,7 @@ class Row extends React.PureComponent {
                 />
               </HoverMenu>
             )}
-            <div
+            <GridRow
               className={cx(
                 'grid-row',
                 rowItems.length === 0 && 'grid-row--empty',
```

Comment:
```suggestion
            <GridRow
              className={cx(
                rowItems.length === 0 && 'grid-row--empty',
```

Reply:
We're using a `grid-row:after` selector in `DashboardBuilder`